### PR TITLE
Add info on command help

### LINKED ISSUE #1384: Maybe add a section on command line help?
### Issue description

It might be helpful to add a note on the Intro to the Command Line page that explains how to get help on commands.

For example, on Windows, most commands have a /? option that gives you help.
On Linux (and Mac?), most have a -h or --help option.

After giving an example (maybe on the delete commands?), it could also be added to the command summary at the end of the page.

### PR DESCRIPTION
**Pull request by jhodgdon-drp**
Fixes #1384

### CODE REVIEW

File: en/intro_to_command_line/README.md [hunk 1]
```diff
@@ -146,6 +146,24 @@ You'll probably see something similar on your machine. Once you open the command
 
 ---
 
+### Learn more about a command
+
+Many commands you can type at the command prompt have built-in help that you can display and read! For example, to learn more about the Current directory command:
```
Comment (das-g) [inline en/intro_to_command_line/README.md]: Is "current" capitalized here intentionally?

File: en/intro_to_command_line/README.md [hunk 2]
```diff
@@ -146,6 +146,24 @@ You'll probably see something similar on your machine. Once you open the command
...
+
+<!--sec data-title="Command help: OS X and Linux" data-id="OSX_Linux_pwd" data-collapse=true ces-->
+
+OS X and Linux have a `man` command, which gives you help on commands. Try `man pwd` and see what it says, or put `man` before other commands to see their help.
```
Comment (das-g) [inline en/intro_to_command_line/README.md]: If the man page doesn't fit in the command line window, `man` will automatically invoke a pager (usually `less`) to display the content. It should probably be mentioned how to navigate in there (arrow keys, page-up/page-down keys) and how to exit the pager (`q` for "quit").

Comment (jhodgdon-drp) [reply en/intro_to_command_line/README.md]: Good idea! I'll update the text...

File: en/intro_to_command_line/README.md (hunk 1 above)
Comment (jhodgdon-drp) [reply en/intro_to_command_line/README.md]: I had thought the tutorial capitalized it but I think you are right, I was looking at a header... will fix.

**Comment by jhodgdon-drp** [comment]
How's this version? Thanks for the feedback!

**Pull request merged**